docs/numerical_exp_report.tex

```diff
@@ -137,291 +135,472 @@ \subsection{Tick}
+\hline
+Cycle Time & 1 & timesteps \\
+Refuel Time & 0 & timesteps \\
+Lifetime & 1 & timesteps \\
+Power Capacity & 1& MWe \\
+Assembly Size & 1 & kg \\
+\# assemblies per core & 1 & \\
+\# assemblies per batch & 1 & \\
+Input Commodity & fuel & kg\\
+Output Commodity & power & MW\\
+\hline
+\textbf{Sink Parameters} & \textbf{Value} & \textbf{Units} \\
+Throughput & 1 & kg \\
+Input Commodity & spent uox & kg\\
+\hline
+	\end{tabular}
 \end{table}
 
-Table \ref{tab:simulationparameters} provides the input and output commodity amounts and enrichment SWUs required based on a reactor fuel demand of 100 kg for one time step. These input and output commodities are based on the simple demand flow of materials shown in figure \ref{diag:dem}. 
+\pagebreak
+
+\section{Numerical Tests for the Non-optimizing prediction method}
+To ensure that the non-optimizing prediction model is working correctly, a variation of input files that uses the non-optimizing method archetype to deploy or decommission facilities is tested to determine if their outputs match the analytical solution. In this section, the tests that must be met is described based on the parameters defined in table \ref{tab:testscenario} and \ref{tab:reactor} and analytical solution of a defined simple scenario. Unit test examples are included in Appendix B.
+
+The tests are split into test A types and test B types. Test A refers to the test scenarios where facilities are expected to be deployed. Test B refers to the test scenarios where facilities are expected to be decommissioned. 
+
+The prediction algorithm for the non-optimizing method has three user-defined input functionalities. The aim with the various test scenarios are to check if the non-optimizing method archetype will deploy or decommission facilities correctly when there is a variation in the combination of the three input functionalities.  
+\begin{enumerate}
+	\item There is an initial demand. 
+	\item There is an initial facility already present. 
+	\item There is a growth in the demand.
+\end{enumerate}
+
+The growth in demand is governed by Equation \ref{eq:growth}. 
+
+\begin{align}
+\label{eq:growth}
+D_f(timestep) &= D_i(1+g)^{(\frac{timestep}{12})}
+\end{align}
+Where $D_f$ is demand of resource at specific time step, $D_i$ is initial demand and g is growth rate. 
+
+The functionalities are tested alone, but also mixed and matched to give various test scenarios to determine if the non-optimizing method archetype will deploy or decommission facilities correctly in the presence of more than one functionality. As stated in the previous section, \texttt{source}, \texttt{reactor} and \texttt{sink} facilities are used in the test scenarios. The test scenarios A1 to A4 and B1 to B2 will only have a \texttt{source} facility, test scenarios A5 to A7 will have both \texttt{source} and \texttt{reactor} facilities and test scenarios A8 and A9 will have both \texttt{source} and \texttt{sink} facilities.
```

Comment:
By `funcionality`, do you mean facilities?

Reply:
I meant parameters, will change the report to reflect it.